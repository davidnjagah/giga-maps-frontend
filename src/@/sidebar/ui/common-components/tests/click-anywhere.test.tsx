import { render, renderHook } from '@testing-library/react';
import ClickAnywhere from '../click-anywhere';
import { testWrapper } from '~/tests/jest-wrapper';

describe('Click anywhere', () => {
  it('should not call insideClick when clicking on an element without the specified class', () => {
    const { asFragment } = render(testWrapper(<ClickAnywhere classList={['sidebar-searchbox']} trigger={true} />));
    expect(asFragment).toMatchSnapshot();
  });
});
