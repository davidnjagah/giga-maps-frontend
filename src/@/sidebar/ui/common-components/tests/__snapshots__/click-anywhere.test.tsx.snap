// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Click anywhere should not call insideClick when clicking on an element without the specified class 1`] = `[Function]`;
